plenum/server/quota_control.py

```diff
@@ -1,40 +1,27 @@
+from stp_zmq.zstack import Quota
+
 
 class QuotaControl:
     def __init__(self,
-                 max_node_message_count_limit,
-                 max_node_message_size_limit,
-                 max_client_message_count_limit,
-                 max_client_message_size_limit):
-        self._max_node_message_count_limit = max_node_message_count_limit
-        self._max_node_message_size_limit = max_node_message_size_limit
-        self._max_client_message_count_limit = max_client_message_count_limit
-        self._max_client_message_size_limit = max_client_message_size_limit
-        self._node_quota_reached = False
-
-    def received_node_messages(self, count: int, size: int):
-        self._node_quota_reached = \
-            count >= self._max_node_message_count_limit or \
-            size >= self._max_node_message_size_limit
-
-    def received_client_messages(self, count: int, size: int):
-        pass
+                 dynamic: bool,
+                 max_request_queue_size: int,
+                 max_node_quota: Quota,
+                 max_client_quota: Quota):
+        self._dynamic = dynamic
+        self._max_request_queue_size = max_request_queue_size
```

Comment:
Do we need the prefix `max` here? `quota` already means that this is some max limit.

Reply:
This is max quota which this strategy will ever allow, so I believe max is appropriate here.